docs/cli.rst

```diff
@@ -121,3 +121,10 @@ By default, ``towncrier`` compares the current branch against ``origin/main`` (a
    Use ``REMOTE-BRANCH`` instead of ``origin/main``::
 
       $ towncrier check --compare-with origin/trunk
+
+.. option:: --staged
+
+   Include files that have been staged for commit when checking for news fragments
```

Comment:
You need double colons to make the following paragraph a [code sample](https://docutils.sourceforge.io/docs/user/rst/quickstart.html#preformatting-code-samples) (otherwise it's a block quote):

```suggestion
   Include files that have been staged for commit when checking for news fragments::
```